thirdeye/thirdeye-frontend/app/shared/filterBarConfig.js

```diff
@@ -13,7 +13,7 @@
 export default [
   {
     header: "Holidays",
-    eventType: "holiday",
+    eventType: "HOLIDAY",
```

Comment:
eventType is used for URN construction and should be lower case

Reply:
Fixed